hash_bcrypt.go

```diff
@@ -34,6 +34,9 @@ type BCrypt struct {
 }
 
 func (b *BCrypt) Hash(ctx context.Context, data []byte) ([]byte, error) {
+	if b.WorkFactor == 0 {
+		b.WorkFactor = bcrypt.DefaultCost
```

Comment:
Since we're defaulting to [`12`](https://github.com/ory/fosite/blob/master/compose/config.go#L111) I think this should also be 12. Potentially as a shared `const`

Reply:
I was thinking of that too. Where should that live?